### PR TITLE
add lemp and postfix roles

### PR DESCRIPTION
**Pull request by lukechinworth**
the gd and image magick were already in the php role so i left that out

### CODE REVIEW

File: ansible/common/postfix/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+- name: Set postfix option hostname
+  debconf:
+    name: postifx
+    question: "postfix/mailname"
+    value: "{{ postfix.mailname }}"
```
Comment (pajtai) [inline ansible/common/postfix/tasks/main.yml:6]: Please add these options and the postfix and lemp tasks to the README.

**Review by pajtai — COMMENTED** [review]
Please also bump the feature flag in package.json.

File: README.md [hunk 2]
```diff
@@ -115,6 +115,12 @@ Some of these depend on each other
           backup_password: "sample backup password"
           version: "3.2" # can be 3.0 or 3.2
         ```
+* LEMP
+* postfix
+    ```
+    postfix:
+        mailname: "{{ ansible_hostname }}"
```
Comment (pajtai) [inline README.md]: What about `postfix.main_mailer_type`?

Comment (pajtai) [inline README.md:122]: What about `postfix.main_mailer_type`?

Comment (lukechinworth) [reply README.md:122]: It looked like you didn't document here in the reader vars in defaults so I left it out. I can add it if you like.

Comment (pajtai) [reply README.md:122]: Please do, vars in defaults are the ones the end user can modify, so it's useful.

**Comment by pajtai** [comment]
merging this, because it is blocking use of postfix, but @lukechinworth please update readme

**Pull request merged**